src/Tools/dotnet-monitor/ParameterCapturing/ParameterCapturingEvents.cs

```diff
@@ -67,12 +67,14 @@ public static class CapturedParametersStartPayloads
         {
             public const int RequestId = 0;
             public const int CaptureId = 1;
-            public const int ActivityId = 2;
-            public const int ActivityIdFormat = 3;
-            public const int ThreadId = 4;
-            public const int MethodName = 5;
-            public const int MethodModuleName = 6;
-            public const int MethodDeclaringTypeName = 7;
+            public const int MethodToken = 2;
```

Comment:
This is an interesting scenario to consider for situations where the inproc component stays the same while dotnet-monitor changes. I don't think we have to worry about that yet, but it seems like it would be possible to make this non-breaking change if we wanted to.

Reply:
See https://github.com/dotnet/dotnet-monitor/pull/7974#discussion_r1975912301 -- this is an intention decision and inline with past decisions around this (https://github.com/dotnet/dotnet-monitor/pull/7493#discussion_r1799777520)